src/content/updates/v10-migration/developers.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: v10 Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [Repo architecture](#repo-architecture)
+- [Migration tooling](#migration-tooling)
+- [Migration strategy](#migration-strategy)
+- [Carbon components](#carbon-components)
+- [Carbon elements](#carbon-elements)
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Repo architecture
+
+Two of Carbon's previous repos will be deprecated with v10: carbon-icons and carbon-themes. Iconography and themes in v10 are now handled as packages in the carbon-elements repo. 
+
+[@carbon/icons](https://github.com/IBM/carbon-elements/tree/master/packages/icons)
+
+[@carbon/themes](https://github.com/IBM/carbon-elements/tree/master/packages/themes)
+
+## Migration tooling
+
+**Coming soon!** [carbon-upgrade](https://github.com/carbon-design-system/carbon-upgrade/blob/master/README.md) is a new tool to help developers upgrade to v10 through the command line. This tool is still experimental, and if you would like to contribute, the Carbon team would welcome the help! [Please leave ideas for migration tooling here](https://github.com/carbon-design-system/carbon-upgrade/issues/7). 
+
+## Migration strategy
```

Comment:
This is the third Job to be Done: Migrate and existing product/project/codebase from v9 components to v10. it's not just developers though - it's a team process. Developers, Designers, OM. likely goes like this:
0) plan
1) build our product w v10
2) generate new design specs to address alignment/sizing changes
3) prioritize order/scope of changes
4) don't mix Design Languages

Reply:
Page has been revised based on this feedback. Thanks, Chris.